commands.js

```diff
@@ -1588,6 +1591,7 @@ var commands = exports.commands = {
 
 		this.logEntry(user.name + " used /endemergency");
 	},
+	emergencyhelp: ["/emergency - Puts the server into emergency mode, making some features limited until a fix. Requires: ~"],
```

Comment:
`emergency` is a (mostly) deprecated function. We shouldn't document it imo, though maybe others are of different opinion?